core/test/Store.js

```diff
@@ -0,0 +1,66 @@
+const { didContractThrow } = require("../../common/SolidityTestUtils.js");
+
+const ERC20MintableData = require("openzeppelin-solidity/build/contracts/ERC20Mintable.json");
+const truffleAssert = require("truffle-assertions");
+const truffleContract = require("truffle-contract");
+const ERC20Mintable = truffleContract(ERC20MintableData);
+ERC20Mintable.setProvider(web3.currentProvider);
+const Store = artifacts.require("Store");
+
+contract("Store", function(accounts) {
+  // A deployed instance of the Store contract, ready for testing.
+  let store;
+
+  const owner = accounts[0];
+  const derivative = accounts[1];
+  const erc20TokenOwner = accounts[2];
+
+  const identifier = web3.utils.utf8ToHex("id");
+
+  //Add test final fee for test identifier
+
+  beforeEach(async function() {
+    store = await Store.new();
+  });
+
+  it("Compute fees", async function() {
+    // Set fee to 10%
+    let newFee = { value: web3.utils.toWei("0.1", "ether") };
+    let txReciept = await store.setFixedOracleFeePerSecond(newFee, { from: owner });
```

Comment:
Is this receipt being used anywhere? Usually you'd only need to use a receipt in a test to check that a particular event fired.